src/app/common/SummaryView.vue

```diff
@@ -41,6 +41,31 @@ const emit = defineEmits<{
 </script>
 
 <style lang="scss" scoped>
+.summary-slideout :deep(.app-view-title-bar) {
+  display: flex;
+  // Accounts for the absolutely-positioned close button
+  margin-right: calc($kui-space-30 + 24px);
+  h1, h2, h3, h4, h5, h6 {
+    --icon-before: url('@/assets/images/icon-wifi-tethering.svg');
```

Comment:
**Question**: Why are you changing how these icons are added to the summary views? I much prefer slotting in the icons instead of this. This generally seems like more work for no benefit.

Reply:
From the PR description:

> The title of the drawer would sometimes start flush-left then move over a little bit as the inline image loaded in, which can be a little jarring. I've seen this happen sometimes when first loading a summary view and sometimes when subsequently clicking on summary views due to Vue re-rendering the DOM and trying to re-load in the inline img. Using CSS avoids all this, and it also reduces a little duplication as we can specify this centrally in the summary view.

Reply:
I’ve never seen that and I can’t reproduce this kind of effect when I try it now.

The slotted content wouldn’t rerender when clicking on various outbound summaries (for example) either because the summary component remains in view.

This doesn’t seem necessary to me.

Reply:
https://github.com/kumahq/kuma-gui/assets/554604/2b69dacd-c1c1-4bed-8709-67392d9af871

Watch the `transmitter-0` text "stutter". The above is locally, I would guess it would be worse if Kuma was on a server somewhere. Ideally I want to inline the svgs also, but didn't figure how to do that with vite just yet.

Granted its a small thing, but its an improvement nonetheless, and all the small things eventually add up.

Reply:
Is there anything else with this PR, or is this the only thing?

Reply:
This issue is quite hard to notice even when focusing on the exact spot where it happens. I don’t see how this would be more pronounced on a poor connection when the bottle neck isn’t download speed of assets but something between the parsing of JS and encoding of the image.

There are a number of ways to tackle this and this change here kind of moves away from using inline SVGs instead of images with data URIs (by moving it from the HTML into the CSS). Providing the summary view with a default image via a slot fallback would maintain the same flexibility while also reducing the flexibility.

In any case, I leave it up to you to improve upon this at a later point as it’s _technically_ working.

Reply:
Oh weird, let me look again before I merge this then..

![Screenshot 2024-02-14 at 13 11 49](https://github.com/kumahq/kuma-gui/assets/554604/6e8c71d2-59ca-4d5c-8daf-8a5c5c72a696)

That is what I'm seeing in prod _previous to this PR_, are you not seeing the same, ah maybe you mean _after this PR_?

Reply:
```sh
cd kuma
git pull
make dev/tools
make build
./build/artifacts-linux-amd64/kuma-cp/kuma-cp run
```

On http://localhost:5681/gui/, the images look like this:

```html
<img data-v-350f9448="" src="data:image/svg+xml,%3csvg%20width='17'%20height='16'%20xmlns='http://www.w3.org/2000/svg'%3e%3cmask%20id='mask0_1444_15097'%20style='mask-type:alpha'%20maskUnits='userSpaceOnUse'%20x='0'%20y='0'%20width='17'%20height='16'%3e%3crect%20x='0.400024'%20width='16'%20height='16'%20fill='%23d9d9d9'/%3e%3c/mask%3e%3cg%20mask='url(%23mask0_1444_15097)'%3e%3cpath%20d='M3.8%2013.5C3.16667%2012.8889%202.66389%2012.1694%202.29167%2011.3417C1.91945%2010.5139%201.73334%209.62222%201.73334%208.66667C1.73334%207.74444%201.90834%206.87778%202.25834%206.06667C2.60834%205.25556%203.08334%204.55%203.68334%203.95C4.28334%203.35%204.98889%202.875%205.8%202.525C6.61111%202.175%207.47778%202%208.4%202C9.32223%202%2010.1889%202.175%2011%202.525C11.8111%202.875%2012.5167%203.35%2013.1167%203.95C13.7167%204.55%2014.1917%205.25556%2014.5417%206.06667C14.8917%206.87778%2015.0667%207.74444%2015.0667%208.66667C15.0667%209.62222%2014.8806%2010.5167%2014.5083%2011.35C14.1361%2012.1833%2013.6333%2012.9%2013%2013.5L12.0667%2012.5667C12.5778%2012.0778%2012.9833%2011.4972%2013.2833%2010.825C13.5833%2010.1528%2013.7333%209.43333%2013.7333%208.66667C13.7333%207.17778%2013.2167%205.91667%2012.1833%204.88333C11.15%203.85%209.88889%203.33333%208.4%203.33333C6.91111%203.33333%205.65%203.85%204.61667%204.88333C3.58334%205.91667%203.06667%207.17778%203.06667%208.66667C3.06667%209.43333%203.21667%2010.15%203.51667%2010.8167C3.81667%2011.4833%204.22778%2012.0611%204.75%2012.55L3.8%2013.5ZM5.68334%2011.6167C5.29445%2011.25%204.98334%2010.8139%204.75%2010.3083C4.51667%209.80278%204.4%209.25556%204.4%208.66667C4.4%207.55556%204.78889%206.61111%205.56667%205.83333C6.34445%205.05556%207.28889%204.66667%208.4%204.66667C9.51111%204.66667%2010.4556%205.05556%2011.2333%205.83333C12.0111%206.61111%2012.4%207.55556%2012.4%208.66667C12.4%209.25556%2012.2833%209.80556%2012.05%2010.3167C11.8167%2010.8278%2011.5056%2011.2611%2011.1167%2011.6167L10.1667%2010.6667C10.4444%2010.4111%2010.6639%2010.1111%2010.825%209.76667C10.9861%209.42222%2011.0667%209.05556%2011.0667%208.66667C11.0667%207.93333%2010.8056%207.30556%2010.2833%206.78333C9.76111%206.26111%209.13334%206%208.4%206C7.66667%206%207.03889%206.26111%206.51667%206.78333C5.99445%207.30556%205.73334%207.93333%205.73334%208.66667C5.73334%209.06667%205.81389%209.43611%205.975%209.775C6.13612%2010.1139%206.35556%2010.4111%206.63334%2010.6667L5.68334%2011.6167ZM8.4%2010C8.03334%2010%207.71945%209.86944%207.45834%209.60833C7.19723%209.34722%207.06667%209.03333%207.06667%208.66667C7.06667%208.3%207.19723%207.98611%207.45834%207.725C7.71945%207.46389%208.03334%207.33333%208.4%207.33333C8.76667%207.33333%209.08056%207.46389%209.34167%207.725C9.60278%207.98611%209.73334%208.3%209.73334%208.66667C9.73334%209.03333%209.60278%209.34722%209.34167%209.60833C9.08056%209.86944%208.76667%2010%208.4%2010Z'%20fill='%236f7787'/%3e%3c/g%3e%3c/svg%3e">
```

Reply:
Ah! I wonder if its a vite upgrade that did that 🤔 we didn't do that until after 2.6 was released, one sec lemme rebuild post-2.6. I'm looking at `2.6.0`

Reply:
Ah yes, this must have been the vite upgrade 🎉 . I guess this is one thing that the vite upgrade gives us 😆 

I wonder if my inlining of CSS SVGs will work now also 🤔 👀 I couldn't get it to work with previous `vite`

Reply:
Ok so that was interesting, here's what I found out (take into account this PR started before we did the vite upgrade):

1. Using a released Kuma 2.6.0 binary (i.e. before we upgraded vite, which was when this PR started). The `<img />` tag src's _do not_ get inlined when built for production.
2. I also noticed when I was working on this PR that I could not get vite to inline my SVGs (which is what I originally wanted)
3. After our `vite` upgrade if you rebuild the kuma binary then the `<img />` tag src's _do_ get inlined.
4. After the vite upgrade SVGs also get inlined 🎉 but vite does not encode the `'`s in any way, so `url('encoded-svg-with-single-quote-unescaped')` does not work. Its also awkward due to double quotes for `style=""`.

To resolve this I moved the CSS to a scoped style, which also lets me get rid of an import which is no longer necessary. This means the unescaped `'`s in the inline style is no longer a problem.

Reply:
> 4\. After the vite upgrade SVGs also get inlined 🎉 but vite does not encode the `'`s in any way, so `url('encoded-svg-with-single-quote-unescaped')` does not work. Its also awkward due to double quotes for `style=""`.

This sounds a lot like https://github.com/vitejs/vite/pull/14643#issuecomment-1904296302, right?

Reply:
Yup, sounds like that at least